src/app/pages/apps/components/available-apps/app-resources-card/app-resources-card.component.ts

```diff
@@ -1,12 +1,58 @@
-import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
-import { Observable } from 'rxjs';
+import {
+  ChangeDetectionStrategy, ChangeDetectorRef, Component, Input, OnInit,
+} from '@angular/core';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { Observable, map, throttleTime } from 'rxjs';
+import { GiB } from 'app/constants/bytes.constant';
+import { MemoryStatsEventData } from 'app/interfaces/events/memory-stats-event.interface';
+import { AvailableAppsStore } from 'app/pages/apps/store/available-apps-store.service';
+import { WebSocketService } from 'app/services';
 
+@UntilDestroy()
 @Component({
   selector: 'ix-app-resources-card',
   templateUrl: './app-resources-card.component.html',
   styleUrls: ['./app-resources-card.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class AppResourcesCardComponent {
+export class AppResourcesCardComponent implements OnInit {
   @Input() isLoading$: Observable<boolean>;
+  cpuPercentage = 0;
+  memoryUsed: string;
+  memoryTotal: string;
+  pool: string;
+
+  constructor(
+    private ws: WebSocketService,
+    private cdr: ChangeDetectorRef,
+    protected applicationsStore: AvailableAppsStore,
+  ) {}
+
+  ngOnInit(): void {
+    this.ws.subscribe('reporting.realtime').pipe(
+      map((event) => event.fields),
+      throttleTime(500),
```

Comment:
Maybe let's even throttle to 1 or 2 seconds. Updating number is a bit distracting.